python-threatexchange/threatexchange/extensions/vpdq/vpdq_util.py

```diff
@@ -13,6 +13,8 @@
 VPDQ_TIMESTAMP_PRECISION = 3
 VPDQ_QUALITY_THRESHOLD = 50
 VPDQ_DISTANCE_THRESHOLD = 31
+VPDQ_QUERY_MATCH_THRESHOLD = 80.0
+VPDQ_INDEX_MATCH_THRESHOLD = 80.0
```

Comment:
blocking q: I think our default in production is 0.0. 

What is your thought about using 80 here?

Reply:
I will replace the default threshold as 0 but still keeping this as an option to filter out low percentage matches.